packages/big-design/src/components/ProgressCircle/ProgressCircle.tsx

```diff
@@ -71,21 +55,23 @@ export class ProgressCircle extends React.PureComponent<ProgressCircleProps> {
         aria-valuetext="Error"
         color="danger"
         role="progressbar"
-        size={this.getDimensions()}
+        size={getDimensions()}
       />
     );
-  }
+  };
 
-  renderSuccess() {
+  const renderSuccess = () => {
     return (
```

Comment:
🍸 I think you could return `CheckCircleIcon ` immediately. `const renderSuccess = () => (...);`